tests/compilers/imports.rs

```diff
@@ -14,15 +14,18 @@ use wasmer::*;
 
 fn get_module(store: &Store) -> Result<Module> {
     let wat = r#"
-        (import "host" "0" (func))
+        (type (func))
+        (import "host" "0" (func $host_func_0 (type 0)))
```

Comment:
Will this ever actually be executed by a test?
Apparently not in the same file.

Would be good to have a dedicated test for this exact issue, which also executes the code.

Reply:
Oh wait... I assumed the module would instantiate

Reply:
https://github.com/wasmerio/wasmer/blob/76b5870702f6535369cb09f56245eb41b726cc2b/tests/compilers/imports.rs#L54

This executes the module afaiu

Reply:
Ideally would also use a host function that returns an integer so the test makes sure it actually is executed correctly.